sdk/core/azure-core-amqp/inc/azure/core/amqp/management.hpp

```diff
@@ -164,7 +172,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
         std::string const& operationToPerform,
         std::string const& typeOfOperation,
         std::string const& locales,
-        Azure::Core::Amqp::Models::AmqpMessage const& messageToSend,
+        Azure::Core::Amqp::Models::AmqpMessage messageToSend,
```

Comment:
```suggestion
        Models::AmqpMessage messageToSend,
```

Question 2: should it be `const&`?

Reply:
It cannot be const& because we modify it. So we need a copy of the message provided by the caller.

I could pass it as a const& and then copy it to another local message, but that is exactly what happens when you specify the parameter as being passed by value.